composer.json

```diff
@@ -20,24 +20,27 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "7.3.99"
+            "php": "7.4.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0"
+        "php": "^7.4 || ~8.0.0 || ~8.1.0"
     },
     "require-dev": {
-        "container-interop/container-interop": "^1.1",
+        "psr/container": "^1.1.2 || ^2.0.2",
         "laminas/laminas-coding-standard": "~2.2.1",
         "laminas/laminas-stdlib": "^3.6",
         "phpbench/phpbench": "^1.1",
         "phpspec/prophecy-phpunit": "^2.0",
         "phpunit/phpunit": "^9.5.5"
     },
     "suggest": {
-        "container-interop/container-interop": "^1.1, to use the lazy listeners feature",
+        "psr/container": "^1.1.2 || ^2.0.2, to use the lazy listeners feature",
```

Comment:
I do not really like the constraint annotation. Lets take the `stdlib` as an example. We suggest using `stdlib` v2 but we do not even test it anymore (due to the fact it was somewhen removed).
Is this some kind of best practice or just done here to keep it in sync with `stdlib`?